src/zc/table/fieldcolumn.py

```diff
@@ -85,7 +89,7 @@ def __init__(self, field, title=None, name=''):
             name = self.field.__name__
         super(FieldColumn, self).__init__(title, name)
 
-    ###### subclass helper API (not expected to be overridden) ######
+    # subclass helper API (not expected to be overridden) ######
```

Comment:
If you're removing the `#####` decoration from the front of the comment, remove it from the back as we..